### PR TITLE
Add caption length validation

### PR DESCRIPTION
**Pull request by omarmaher100**
Hi Geeks
I want to add validation on caption length because the maximum length of it on telegram server is 200 character

### CODE REVIEW

File: TeleSharp.TL/TL/TLInputMediaUploadedDocument.cs [hunk 1]
```diff
@@ -22,7 +22,8 @@ public override int Constructor
      public TLAbsInputFile file {get;set;}
      public string mime_type {get;set;}
      public TLVector<TLAbsDocumentAttribute> attributes {get;set;}
-     public string caption {get;set;}
+        private string caption;
```
Comment (knocte) [inline TeleSharp.TL/TL/TLInputMediaUploadedDocument.cs]: Wrong indentation, please use same indentation as before.

File: TeleSharp.TL/TL/TLInputMediaUploadedDocument.cs [hunk 2]
```diff
@@ -22,7 +22,8 @@ public override int Constructor
...
+     public string Caption { get { return caption; } set { caption = value.Length > 200 ? value.Remove(199, value.Length - 199) : value; } }
```
Comment (knocte) [inline TeleSharp.TL/TL/TLInputMediaUploadedDocument.cs]: This line being so long is not readable, please split in many lines.

**Comment by knocte** [comment]
Where is the maximum length documented?

**Comment by knocte** [comment]
Why do you close the PR? You can change your commits without the need to create a new PR.

**Comment by omarmaher100** [comment]
sorry i don't know how :)

**Pull request closed without merging**